src/main/java/org/embulk/input/marketo/delegate/ProgramMembersBulkExtractInputPlugin.java

```diff
@@ -214,6 +214,9 @@ public TaskReport ingestServiceData(final PluginTask task, RecordImporter record
                 }
             }
             restClient.close();
+            if (executor != null && !executor.isShutdown()) {
+                executor.shutdownNow();
+            }
```

Comment:
Should be in `try-finally` to make sure any exception throws above this ThreadExecutor will stopped.